src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/softmax.hpp

```diff
@@ -15,7 +15,7 @@ using SoftMaxTestParams = std::tuple<
         ElementType,                                    // inPrecision
         ElementType,                                    // outPrecision
         InputShape,                                     // Dynamic shape + Target static shapes
-        size_t,                                         // axis
+        int64_t,                                        // axis
```

Comment:
Those change in existing structure causes modification of all plugin test instances. CPU and GPU are in github but OpenVINO has VPU/HDDL/ARM and others which are not in github. Please create new structure for Softmax8 and keep existing structure for Softmax1.

Reply:
Done